lib/services/local/users.go

```diff
@@ -578,14 +579,30 @@ func (s *IdentityService) UpsertWebauthnLocalAuth(ctx context.Context, user stri
 		return trace.Wrap(err)
 	}
 
-	value, err := json.Marshal(wla)
+	wlaJSON, err := json.Marshal(wla)
 	if err != nil {
-		return trace.Wrap(err)
+		return trace.Wrap(err, "marshal webauthn local auth")
 	}
-	_, err = s.Put(ctx, backend.Item{
+	if _, err = s.Put(ctx, backend.Item{
 		Key:   webauthnLocalAuthKey(user),
-		Value: value,
+		Value: wlaJSON,
+	}); err != nil {
+		return trace.WrapWithMessage(err, "writing webauthn local auth")
```

Comment:
```suggestion
		return trace.Wrap(err, "writing webauthn local auth")
```

Reply:
Why do you prefer Wrap over WrapWithMessage? I kind of like the clarity of the latter, but either works TBH.

Reply:
Done.